### ISSUE TITLE
Python Support

### ISSUE THREAD

**Issue by braxtonhall** [ataraxie/codeshovel#29]
As seen here #27
Messed up and accidentally merged early.

**Comment by rtholmes** [comment]
@braxtonhall what was the status for the Python support in the end?

**Comment by braxtonhall** [comment]
Oh sorry I missed this message, but it works! There are only 40 unit tests, but it passes 39 of them

**Comment by rtholmes** [comment]
Just reopening. In the merge to bring this up with the changes for icse (all the way to HEAD), the `develop` branch now does not compile because `PythonFunction` needs a `getInitialId` method and I'm not sure what should be there.

**Comment by braxtonhall** [comment]
It compiled for me! Did you run maven again? It should generate a lot of Java code for you that has been ignored

**Comment by rtholmes** [comment]
When I run `mvn compile` on the `develop` branch I get:

```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/rtholmes/dev/codeshovel/src/main/java/com/felixgrund/codeshovel/parser/impl/PythonFunction.java:[18,8] com.felixgrund.codeshovel.parser.impl.PythonFunction is not abstract and does not override abstract method getInitialId(PythonParseTree.PythonParser.FuncdefContext) in com.felixgrund.codeshovel.parser.AbstractFunction
[INFO] 1 error
```

I did also have to resolve some `pom.xml` conflicts too, I wonder if you have some dependencies installed I'm just missing because my `pom.xml` isn't right?

**Comment by braxtonhall** [comment]
OK I'll have a look on Sunday. I think it'll be an easy fix

**Comment by braxtonhall** [comment]
OK so the issue comes from this commit which wasn't in the tree I was testing 🤦‍♀️ 
https://github.com/ataraxie/codeshovel/commit/f14be5c97a99fdbc4937e35653bad8baa6c99d53
Just need to clear up what its purpose is before I fix it #52
